src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -122,6 +122,17 @@ public void remove(ReadOnlyPerson toRemove) throws PersonNotFoundException {
         }
     }
 
+    /**
+     * Sorts the persons in the list
+     */
+    public void sort() {
+        internalList.sort((person1, person2) -> {
+            String name1 = person1.getName().fullName;
```

Comment:
Consider implementing `compareTo` method in Name class or making it implement `Comparable` class, instead of reaching into Name object to get its String value. You can read up on Law of Demeter on this (https://nus-cs2103-ay1819s2.github.io/cs2103-website/se-book-adapted/chapters/principles.html#law-of-demeter).